docker/metadata.json

```diff
@@ -963,6 +963,12 @@
         }
       }
     },
+    "inputs": [
+      {
+        "namespace": "food_delivery",
+        "name": "public.restaurants"
+      }
+    ],
```

Comment:
Not sure we actually want to merge this, but it was useful for repro'ing the issue locally and validating the fix.

Reply:
I think we should remove this before a merge.